### DISCUSSION TITLE
Firefox start time

### DISCUSSION THREAD

**Discussion by JustasPolis** [hyprwm/Hyprland#4384] [Q&A] [answered]
Hey, am I the only one observing long launch times for firefox ? 3-4 seconds for first window launch. Other GTK apps are fine, launched instantly. 

What is interesting, that after first window opens, second window opens instantly.

Mozilla 121.0 version, running wayland mode. 

I though maybe something to do with xdg portals, but uninstalled all portals and I only observe this with mozilla.

**Comment by fufexan** [comment]
Have you tried opening Firefox with a new profile? In many cases, the users' existing profiles are what make Firefox take longer to open.

**Reply by JustasPolis** [reply, marked as answer]
Yeah, seems it's normal for this laptop to take so long to open firefox. 
thanks anyways.
